R/cookies.R

```diff
@@ -130,30 +108,15 @@ cookie_banner_ui <- function(id, name = "DfE R-Shiny dashboard template") {
 #' @export
 #'
 #' @examples
-#' \donttest{
-#' #' To use the banner, first copy the file cookie-consent.js from the
-#' # inst/ folder of this package to the www/ folder of your R-Shiny app.
-#' # Then add the following lines to your ui.R:
-#' \dontrun{dfe_cookie_script(),}
 #' \dontrun{
```

Comment:
Don't mind if you want to pick this up as a separate thing, though for the examples we could probably do a full mini app.R script to give the function in context and then use `if(interactive()){}` instead of `\dontrun`.

The [shinyWidgets package](https://dreamrs.github.io/shinyWidgets/reference/WinBox.html) (lots of others too, this was just the first I googled) has some examples of this

Reply:
Yeah, that makes sense and would be neater than what I just removed! As above, one for another PR I think.

Reply:
Have raised as https://github.com/dfe-analytical-services/dfeshiny/issues/35 for us to come back to later